L10nSnapshotTests/L10nSuite2SnapshotTests.swift

```diff
@@ -112,8 +112,13 @@ class L10nSuite2SnapshotTests: L10nBaseSnapshotTests {
         waitForExistence(app.cells["Logins"], timeout: 15)
         app.cells["Logins"].tap()
 
-        waitForExistence(app.navigationBars.buttons.firstMatch, timeout: 15)
-        app.otherElements.buttons.element(boundBy: 2).tap()
+        // First time only: The message "Your passwords are now protected
```

Comment:
This change only works for me the second time the app is launched. The first time I see this screen and can't tap on the Conrinue button to see the logins view:
![Simulator Screen Shot - iPhone 14 - 2023-03-23 at 14 18 57](https://user-images.githubusercontent.com/1897507/227217304-4d07067f-873c-4e43-bce3-60d6809158ee.png)

Reply:
We need a different approach to be able to click on this screen... not sure how... difficult to deal with these screens that only appears the first time..

Reply:
This is how it is done in the logins tests: 
https://github.com/mozilla-mobile/firefox-ios/blob/4d5684eee0d8e337abf308d99dceab82158a472f/Tests/XCUITests/SaveLoginsTests.swift#L38

Reply:
But I think it was not working perfectly in the past either in these tests...

Reply:
> This change only works for me the second time the app is launched. The first time I see this screen and can't tap on the Conrinue button to see the logins view: ![Simulator Screen Shot - iPhone 14 - 2023-03-23 at 14 18 57](https://user-images.githubusercontent.com/1897507/227217304-4d07067f-873c-4e43-bce3-60d6809158ee.png)

This is what I see on my end on the first time the app is launched

![Simulator Screen Shot - iPhone 14 - 2023-03-22 at 16 37 27](https://user-images.githubusercontent.com/1740517/227244728-d9c1f84f-ca70-4fc0-8ed0-bba3f82c6534.png)

Here's the snippet of `debugDescription`:

```
Other, 0x7f9f8f80df70, {{0.0, 0.0}, {390.0, 844.0}}
        Other, 0x7f9f8f810d20, {{0.0, 0.0}, {390.0, 844.0}}
          NavigationBar, 0x7f9f8f80d490, {{0.0, 47.0}, {390.0, 44.0}}, identifier: 'Passwords'
            Button, 0x7f9f8f80d100, {{0.0, 47.0}, {93.3, 44.0}}, label: 'Settings'
            StaticText, 0x7f9f8f80b4b0, {{152.0, 58.7}, {86.3, 20.3}}, label: 'Passwords'
          Other, 0x7f9f9da1af50, {{0.0, 0.0}, {390.0, 844.0}}
            Other, 0x7f9f9da0dcd0, {{0.0, 0.0}, {390.0, 844.0}}
              Other, 0x7f9f9da19cc0, {{0.0, 0.0}, {390.0, 844.0}}
                StaticText, 0x7f9f9da1a910, {{20.0, 399.3}, {350.0, 45.3}}, label: 'Your passwords are now protected by Face ID, Touch ID or a device passcode.'
                Button, 0x7f9f9da1aac0, {{147.0, 464.7}, {96.0, 35.0}}, label: 'Learn More'
                  StaticText, 0x7f9f9da1a200, {{147.0, 470.7}, {96.0, 23.0}}, label: 'Learn More'
                Button, 0x7f9f9da1a310, {{35.0, 746.0}, {320.0, 44.0}}, label: 'Continue'
                  StaticText, 0x7f9f9da1bb50, {{164.7, 757.0}, {76.0, 22.0}}, label: 'Continue'
```

If there's no other `app.buttons["Continue"]` elsewhere, let me test the existence of this button just in case we run this test for the very first time.